<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopInfo</class>
 <widget class="QMainWindow" name="ShopInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оружейный магазин</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>741</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="pageShop">
     <widget class="QPushButton" name="AddShop">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/new/prefix1/res/w512h5121380984207add.png</normaloff>:/new/prefix1/res/w512h5121380984207add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="RemoveShop">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/new/prefix1/res/w512h5121380984637delete1.png</normaloff>:/new/prefix1/res/w512h5121380984637delete1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdateShop">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить базу</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableViewShop">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>721</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="EditShop">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/new/prefix1/res/edit.png</normaloff>:/new/prefix1/res/edit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Магазин&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageProducts">
     <widget class="QTableView" name="tableViewProduct">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>721</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="RemoveProduct">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/new/prefix1/res/w512h5121380984637delete1.png</normaloff>:/new/prefix1/res/w512h5121380984637delete1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="EditProduct">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/new/prefix1/res/edit.png</normaloff>:/new/prefix1/res/edit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdateProduct">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить базу</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddProduct">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/new/prefix1/res/w512h5121380984207add.png</normaloff>:/new/prefix1/res/w512h5121380984207add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Товары&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pagePersons">
     <widget class="QTableView" name="tableViewPerson">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>721</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="RemovePerson">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/new/prefix1/res/w512h5121380984637delete1.png</normaloff>:/new/prefix1/res/w512h5121380984637delete1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="EditPerson">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/new/prefix1/res/edit.png</normaloff>:/new/prefix1/res/edit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdatePerson">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить базу</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddPerson">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/new/prefix1/res/w512h5121380984207add.png</normaloff>:/new/prefix1/res/w512h5121380984207add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Персонал&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageBuy">
     <widget class="QTableView" name="tableViewBuy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>721</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="RemoveBuy">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/new/prefix1/res/w512h5121380984637delete1.png</normaloff>:/new/prefix1/res/w512h5121380984637delete1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="EditBuy">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Редактировать</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/new/prefix1/res/edit.png</normaloff>:/new/prefix1/res/edit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdateBuy">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить базу</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddBuy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/new/prefix1/res/w512h5121380984207add.png</normaloff>:/new/prefix1/res/w512h5121380984207add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Покупки&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Оружейный магазин</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Оружейный магазин</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShop"/>
   <addaction name="actionProducts"/>
   <addaction name="actionPersons"/>
   <addaction name="actionBuy"/>
  </widget>
  <action name="actionShop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/res/shop.png</normaloff>:/new/prefix1/res/shop.png</iconset>
   </property>
   <property name="text">
    <string>Магазин</string>
   </property>
  </action>
  <action name="actionProducts">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/res/Crime_icon-icons.com_54190.png</normaloff>:/new/prefix1/res/Crime_icon-icons.com_54190.png</iconset>
   </property>
   <property name="text">
    <string>Товары</string>
   </property>
   <property name="toolTip">
    <string>Товары</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPersons">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/res/5.png</normaloff>:/new/prefix1/res/5.png</iconset>
   </property>
   <property name="text">
    <string>Персонал</string>
   </property>
  </action>
  <action name="actionBuy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/res/cart.png</normaloff>:/new/prefix1/res/cart.png</iconset>
   </property>
   <property name="text">
    <string>Покупки</string>
   </property>
   <property name="toolTip">
    <string>Покупки</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
